inc/blocks/namespace.php

```diff
@@ -40,6 +41,10 @@ function setup() {
 		AB_Test_Variant\setup();
 		$using_blocks = true;
 	}
+	if ( Utils\is_feature_enabled( 'broadcast' ) ) {
+		Broadcast\setup();
+		$using_blocks = true;
```

Comment:
This variable should have perhaps been tailored to being about the requirement for the shim block - the shim block is only used for validating blocks that have conversion goals, e.g. checking for links / buttons etc... so I don't think you need it for the broadcast block.